Docker/python-nginx/python3.7-alpine/dockerrun.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
```

Comment:
@johnfrancismccann , please make sure you build a fence and/or indexd image out of this base image AND run a fence and/or indexd test PR to make sure this new base image won't introduce any regressions.
Let me know if you need any help with setting up this test.